src/pages/deduplicate/index.tsx

```diff
@@ -0,0 +1,309 @@
+import EnteSpinner from 'components/EnteSpinner';
```

Comment:
please refactor this whole page and only have whats absolutely necessary

Reply:
I did minimize it to as much possible but I am not sure what else we can remove as all the current components seem necessary.